src/components/Content.jsx

```diff
@@ -1,35 +1,17 @@
-import { useEffect, useState } from "react"
-import { createReviewCards } from "./utils"
-import { getReviews } from "./api";
+import { ReviewList } from './Review-list';
+import { useLocation } from 'react-router-dom';
+import { SingleReview } from './Single-review';
 
 export const Content = () => {
-
-  const [reviews, setReviews] = useState([]);
-  const [isLoading, setIsLoading] = useState(true);
-	const [error, setError] = useState(null);
-
-	useEffect(() => {
-		const fetchData = async () => {
-			try {
-				const data = await getReviews();
-				setReviews(data);
-			} catch (error) {
-				setError('Error fetching data from api');
-			}
-			setIsLoading(false);
-		};
-		fetchData();
-	}, []);
+	let { pathname } = useLocation();
```

Comment:
we shouldn't be using `useLocation()` as this is what `routes` is doing - so this whole content component doesn't need to exist as your routes in App should be rendering either your `Review-list` component or `Single-review` depending on the url.